packages/testing-kit/src/wallet/lucid/index.ts

```diff
@@ -63,7 +63,8 @@ const waitRuntimeSyncingTillCurrentWalletTip =
     await waitForPredicatePromise(
       isRuntimeChainMoreAdvancedThan(client, currentLucidSlot)
     );
-    return sleep(5);
+    process.stdout.write("\n");
+    return sleep(15);
```

Comment:
👁️ ... 
what is this sleep 5 turned into 15?
Why are we sleeping on a random number? Could you add a comment to it?